tools/misc-types/build.gradle.kts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+plugins {
+  alias(libs.plugins.jandex)
+  id("polaris-client")
+}
+
+description =
+  "Misc types used in configurations and converters for microprofile-config & Jackson, exposes no runtime dependencies"
+
+dependencies {
```

Comment:
Does this really require its own build file?

And if we're going to add one with its own dependencies, can't we just pull in a dependency for this? For example, there is `org.apache.commons.io.FileUtils.ONE_MIB`

Reply:
A dependency for a constant feels a bit too much IMO